.renovaterc.json

```diff
@@ -76,6 +76,14 @@
       "paths": ["+(package.json)"],
       "updateTypes": ["minor"],
       "groupName": "core dependencies (minor updates)"
+    },
+
+    {
+      "assignAutomerge": true,
+      "automerge": true,
```

Comment:
We currently use [`forking-renovate`](https://github.com/apps/forking-renovate), which only has read access to code:

![image](https://user-images.githubusercontent.com/26553114/81969950-b9c90180-95ec-11ea-8d28-373adc42b23b.png)


Do you know if auto-merge can work at all with `forking-renovate`? Or does it need us to switch to the more powerful [`renovate`](https://github.com/apps/renovate)?

Reply:
It looks like there's no clean way to make it actually "hit the merge button" in the way we'd like. There's some mention in the doc of bots merging PRs for renovate as well. I'll have to do some digging to figure out what approach is best for automerge, but in the meantime this grouping is still useful.